src/Content/MediaFactory.php

```diff
@@ -60,7 +60,7 @@ public function createOrUpdateMedia(SplFileInfo $file, string $area, ?string $ti
             $media = new Media();
             $media->setFilename($file->getFilename())
                 ->setPath($path)
-                ->setArea($area);
+                ->setArea($fileLocation);
```

Comment:
setLocation

Reply:
Done. I've updated the `Media` Entity, so it'll need a `make db-reset`. :-)